src/components/groupBrowsing/GroupBrowsingCard/GroupBrowsingCard.tsx

```diff
@@ -0,0 +1,255 @@
+import React, { FC } from 'react';
+import { styled } from 'stitches.config';
+import UserIcon from '@assets/svg/user.svg';
+import CalendarIcon from '@assets/svg/calendar.svg';
+import Avatar from '@components/avatar/Avatar';
+import { Flex } from '@components/util/layout/Flex';
+import { fonts } from '@sopt-makers/fonts';
+import { GroupBrowsingCardDetail, parsePartValueToLabel } from '@api/meeting';
+import dayjs from 'dayjs';
+import { ACTION_STATUS, EActionStatus } from '@constants/option';
+import Link from 'next/link';
+
+const GroupBrowsingCard: FC<GroupBrowsingCardDetail> = ({
+  id,
+  userId,
+  title,
+  category,
+  imageURL,
+  startDate,
+  endDate,
+  capacity,
+  mStartDate,
+  mEndDate,
+  recentActivityDate,
+  targetActiveGeneration,
+  joinableParts,
+  applicantCount,
+  approvedUserCount,
+  user,
+  status,
+}) => {
+  const isAllParts = joinableParts?.length === 6 || joinableParts === null;
+  const isGroupActive = new Date(mStartDate) <= new Date() && new Date() <= new Date(mEndDate);
+
+  const returnNewStatus = () => {
+    if (status == 0) {
+      return 0;
+    } else if (status == 1) {
+      return 1;
+    } else {
+      if (new Date(mStartDate) > new Date()) {
+        return 2;
+      } else if (isGroupActive) {
+        return 3;
+      } else {
+        return 4;
+      }
+    }
+  };
+
+  const newStatus = returnNewStatus();
+
+  const returnStatusText = () => {
+    switch (newStatus) {
+      case EActionStatus.BEFORE:
+        return <SStatusText status={newStatus}>{dayjs().diff(startDate, 'day')}일 남음</SStatusText>;
+      case EActionStatus.RECRUITING:
+        return <SStatusText status={newStatus}>{applicantCount}명 신청 중</SStatusText>;
+      case EActionStatus.ACTING:
+        return (
+          <SStatusText status={newStatus}>
+            {recentActivityDate ? `${dayjs().diff(recentActivityDate, 'day')}일 전 활동` : '오늘 새 글'}
+          </SStatusText>
+        );
+    }
+  };
+
+  return (
+    <Link href={`/group/detail?id=${id}`}>
+      <a>
+        <SGroupBrowsingCard>
+          <SInfo>
+            <STop>
+              <Avatar src={user.profileImage} alt="" sx={{ width: 18, height: 18 }} /> {user.name}
+              <STopDivisor>|</STopDivisor> {category}
```

Comment:
```suggestion
              <Avatar src={user.profileImage} alt="" sx={{ width: 18, height: 18 }} />
              <span>{user.name}</span>
              <STopDivisor>|</STopDivisor>
              <span>{category}</span>
```

요런 식으로 user.name과 category에는 span으로 한 번 감싸주는 게 어떨까?
가독성 측면에서 더 좋은 것 같아서 제안!

Reply:
헉 확실히 가독성이 더 좋아진 거 같애!! <3